### ISSUE TITLE
When I scroll up the keyboard goes away.

### ISSUE THREAD

**Issue by ghost** [ish-app/ish#1759]
I have an iPhone 12 running the latest iOS version. I also have voiceover and when I go up or scroll up, the keyboard goes away and I cannot get it back.
